hydroflow/src/compiled/pull/symmetric_hash_join.rs

```diff
@@ -88,6 +88,14 @@ where
     type Item = (Key, (V1, V2));
 
     fn next(&mut self) -> Option<Self::Item> {
+        if self.state.0.table.is_empty() && self.lhs.peek().is_none() {
+            return None;
```

Comment:
with return None; state.x.build() will never get called, so does this produce the correct behavior across ticks for 'static joins?

Reply:
For a 'static join, the table will preserve its values from the previous tick, right? so the table should not be empty? If I understand how this works

Reply:
Yeah, 'static will preserve it.

I understand now, it works because it is peek(), I misread that.